### PR TITLE
Task/20326 rpsec controller

### PR DESCRIPTION
**Pull request by tnminh00**
==> Title: refers task/20326 rspec controller

==> Redmine ticket: https://edu-redmine.sun-asterisk.vn/issues/20326

==> Attachment:

### CODE REVIEW

**Comment by tnminh00** [comment]
ready

File: app/controllers/chapters_controller.rb [hunk 1]
```diff
@@ -38,7 +38,7 @@ def create
     @story = Story.find_by(id: params[:story][:id])
     @chapter = @story.chapters.new chapter_params
 
-    if @story.chapters.where(chapter_number: chapter_params[:chapter_number]).exists?
+    if @story.chapters.where(chapter_number: chapter_params[:chapter_number]).blank?
```
Comment (ngocvt-0484) [inline app/controllers/chapters_controller.rb]: where này viết scope nhé e

**Comment by tnminh00** [comment]
ready

File: app/controllers/chapters_controller.rb [hunk 2]
```diff
@@ -38,7 +38,7 @@ def create
     @story = Story.find_by(id: params[:story][:id])
     @chapter = @story.chapters.new chapter_params
 
-    if @story.chapters.where(chapter_number: chapter_params[:chapter_number]).exists?
+    if @story.chapters.chapter_number(chapter_params[:chapter_number]).blank?
```
Comment (ngocvt-0484) [inline app/controllers/chapters_controller.rb]: đặt tên scope là by_chapter_number r c merge nhé

Comment (tnminh00) [reply app/controllers/chapters_controller.rb]: vâng ạ

**Comment by tnminh00** [comment]
ready

**Pull request merged**